app/src/main/java/com/example/atheneum/activities/AddEditBookActivity.java

```diff
@@ -123,6 +127,43 @@ public void onChanged(@Nullable Book book) {
                     bookLiveData.removeObserver(this);
                 }
             });
+
+            FirstBookPhotoViewModelFactory bookPhotoViewModelFactory = new FirstBookPhotoViewModelFactory(bookID);
+            FirstBookPhotoViewModel bookPhotoViewModel = ViewModelProviders
+                                                            .of(this, bookPhotoViewModelFactory)
+                                                            .get(FirstBookPhotoViewModel.class);
+            bookPhotoViewModel.getBookPhotoLiveData().observe(this, new Observer<Photo>() {
+                @Override
+                public void onChanged(@Nullable Photo photo) {
+                    if (photo != null) {
+                        Glide.with(getApplicationContext())
+                                .load(Photo.DecodeBase64BitmapPhoto(photo.getEncodedString()))
+                                .apply(new RequestOptions()
+                                        .centerCrop()
+                                        .format(DecodeFormat.PREFER_ARGB_8888))
+                                .into(bookImage);
+                    }
+                }
+            });
+
+            bookImage.setOnClickListener(new View.OnClickListener() {
```

Comment:
This on click listener only applies in edit mode. Is there a way to move it out so it also applies when creating a new book? Right now it seems tied to the existing book because of live data.

Reply:
I think the listener only applies in edit mode because of the way the database stuff is done right now. Hmmmm, this might be a tough one....

Reply:
Fixed